.github/workflows/prod-deploy.yaml

```diff
@@ -0,0 +1,42 @@
+name: Deploy to Prod
+
+on: workflow_dispatch
+
+# https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/configuring-openid-connect-in-amazon-web-services
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  deploy-to-prod:
+    concurrency:
+      group: prod-cryoet-api-deploy-${{ github.ref }}
+      cancel-in-progress: true
```

Comment:
Curious why this is here?

Reply:
I retained it from the copy pasta, to handle the case where we accidentally trigger multiple executions of prod deployments manually.